Code/src/main/java/nl/utwente/group10/ui/CustomUIPane.java

```diff
@@ -138,10 +138,11 @@ public Env getEnvInstance() {
     /**
      * Re-evaluate all display blocks.
      */
-    public final void invalidate() {
+    public final void invalidateAll() {
+        System.out.println("Using inefficient invalidation!");
```

Comment:
Als dit echt een probleem is mag je een exception gooien, als het niet echt een probleem is hoef je niks te printen.

Als beide opties je niks lijken heb ik een trucje dat je eventueel kunt doen: deze methode een andere methode met een hele duidelijke naam laten aanroepen: `invalidateEverythingEverywhereInefficiently` of iets dergelijks geks. Als we het geheel dan gaan profilen en het systeem spendeert veel tijd in die methode weten we wat er aan de hand is.

Als we nooit aan profilen toekomen hebben we kennelijk geen performanceprobleem.